src/main/java/mate/academy/spring/service/RoleService.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.spring.service;
+
+import mate.academy.spring.model.Role;
+
+public interface RoleService {
+    Role add(Role role);
+
+    Role getByName(String roleName);
```

Comment:
I guess it will be better to use Optional because we use user input data.
(See userService.findByEmail)